### PR TITLE
Created zoo.js, finished assignment

### PR DESCRIPTION
**Pull request by greenitthe**

### CODE REVIEW

File: zoo.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+              new Animal("Asteria",	"Western lowland gorilla",	"Gorilla gorilla gorilla",	"Fagan Valley",	5,	"Western_lowland_gorilla.jpg"),
+              new Animal("Perses",	"Wolf's mona monkey",	"Cercopithecus wolfi",	"Fagan Valley",	10,	"Wolfs_mona_monkey.jpg"),
+              new Animal("Pip",	"Red river hog",	"Potamochoerus porcus",	"Fagan Valley",	4,	"Red_river_hog.jpg"),
+              new Animal("Skrytnaya",	"Amur leopard",	"Panthera pardus orientalis",	"Kond Cat Complex",	8,	"Amur_leopard.jpg"),
+              new Animal("Matadi",	"African lion",	"Panthera leo",	"Kond Cat Complex",	6,	"African_lion.jpg"),
+              new Animal("Kimani",	"Siberian tiger",	"Panthera tigris altaica",	"Kond Cat Complex",	9,	"Siberian_tiger.jpg"),
+              new Animal("Antipode",	"Polar bear",	"Ursus maritimus",	"Moser Canyon",	11,	"Polar_bear.jpg") );
+
+//Sort the animals alphabetically by name
+animals.sort();
```
Comment (barberboy) [inline zoo.js]: Calling `.sort()` without a compare function will not sort the animals by their first name. You will have to define a compare function and pass it to `.sort()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Description

File: zoo.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  this.commonName = commonName;
+  this.species = species;
+  this.location = location;
+  this.age = age;
+  this.image = image;
+}
+
+//Create the animals... So many animals...
+var animals = [];
+animals.push( new Animal("Mooney", "Blue monkey", "Cercopithecus mitis", "Smith Jungle", 2, "Blue_monkey.jpg"),
```
Comment (barberboy) [inline zoo.js:13]: I don't mind the `Animal` function constructor—it keeps you from having to re-enter the key names (like `{name: 'Mooney', commonName: Blue monkey...}`)—but having so many parameters in a function definition is not ideal. It's kind of a mixed bag.

With JavaScript, it's more common to use an [Object literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer) to define objects than to use constructor functions _unless_ the object has methods.

**Review by barberboy — CHANGES_REQUESTED** [review]
You can decide whether or not to switch to the object literal, but please provide a compare function to the `.sort()` method to sort by the animal name.

**Comment by greenitthe** [comment]
Fixed the issue. Leaving as object constructor since it would be effort to refactor and works as intended without changing. I'll keep using more object literals in mind for the future.

**Review by barberboy — APPROVED** [review]

**Comment by barberboy** [comment]
👍 💯

**Pull request closed without merging**